lib/routes/reuters/channel.js

```diff
@@ -85,8 +85,37 @@ module.exports = async (ctx) => {
                 default:
                     break;
             }
+        } else if (site === 'us') {
+            linkSelector = '.story-content a';
+            switch (channel) {
+                case 'business':
+                    title += 'Business';
+                    link = homePage;
+                    break;
+                case 'world':
+                    title += 'World';
+                    break;
+                case 'uk':
+                    title += 'UK';
+                    break;
+                case 'technology':
+                    title += 'Tech';
+                    break;
+                case 'personalFinance':
+                    title += 'Money';
+                    link = 'https://uk.reuters.com/business';
+                    break;
+                case 'breakingviews':
+                    title += 'Breakingviews';
+                    link = homePage;
+                    break;
+                case 'lifestyle':
+                    title += 'Life';
+                    break;
+                default:
+                    break;
```

Comment:
Are you sure this section is correct?

Reply:
Well... no. I was asking should we simply allow the user to enter categories because new categories are quite actively invented. Since you're concern of whether our current rules apply for all categories, I'll look into it. Most pages are fine. But I do find [at least one exception](https://www.reuters.com/finance/summits). Maybe this instance could be handled as an exception, whereas we allow to use the same path where the page has a `.story-content a` feed.

Reply:
@HenryQW What do you think of it?

Reply:
That's why I only hardcode channels from the top-level navibar, for this version I think it will suffice.![image](https://user-images.githubusercontent.com/5896343/86808058-6a460580-c072-11ea-9462-a9b5efded660.png) So for now can you pls hardcode them?

We can expand further to support all sub-channel (those in dropdowns of the top-level navibar) or tags in the future. They will most likely use another route like `/reuters/tag/:site/:tag`.

Also after a quick check, only US and UK sites are sharing the selector, for other sites I don't think RSSHub has the user base so it will be supported when requested.

Reply:
> So for now can you pls hardcode them?
> 
> OK.